roles/os_zabbix/vars/template_openshift_master.yml

```diff
@@ -303,6 +328,14 @@ g_template_openshift_master:
     description: The application create loop has failed 4 or more times in the last hour
     priority: avg
 
+  - name: 'Application with build creation has failed multiple times in the last hour on {HOST.NAME}'
+    expression: '{Template Openshift Master:openshift.master.app.build.create.sum(1h)}>3'
```

Comment:
The cron job runs every 30 minutes.  Therefore this trigger will never fail more than 3 times an hour.
Either change this to '{Template Openshift Master:openshift.master.app.create.sum(2h)}>3'.  The problem with this is that we want to know that it is slow before the 1.5 hour mark.

Or update the cron to run every 10-15 minutes and then we can see if it fails more than 3 times.



 */30 * * * * export ZAGG_CLIENT_HOSTNAME=ops-master-f58e0; ops-runner -f -s 15 -n csca.openshift.master.app.build.create /usr/bin/cron-send-build-app